### PR TITLE
refactor(android): extract action & value strings as static constants

### PR DESCRIPTION
**Pull request by erisu**
### Platforms affected

android

### Motivation and Context

Extract action and value strings as constants.

### Description

Created `private static` variables of the actions and value strings and hoisted to top.

### Testing

build test within AS

### Checklist

- [ ] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [x] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

### CODE REVIEW

File: src/android/StatusBar.java [hunk 1]
```diff
@@ -36,12 +36,24 @@
 import org.apache.cordova.CordovaWebView;
 import org.apache.cordova.LOG;
 import org.apache.cordova.PluginResult;
+import org.jetbrains.annotations.NonNls;
```
Comment (breautek) [inline src/android/StatusBar.java:39]: is this import really safe? Is it included in the Android SDK?

Comment (breautek) [reply src/android/StatusBar.java:39]: On a basic gradle project, this import fails, but in a cordova-android project this import works as is.

But `org.jetbriains` is not existent in the Android documentation, and I don't see how we are importing `'org.jetbrains:annotations:23.0.0'` (via [jetbrains documentation](https://github.com/JetBrains/java-annotations#using-the-annotations)) so this import might be relying on a third-party dependency implementation detail.

I wonder if we should be explicitly declaring this dependency in gradle.

Comment (erisu) [reply src/android/StatusBar.java]: I went ahead and removed the import and annotations usage of `NonNls`.
I added it only to remove the IDEs annoyance for localization on these items.
It is not necessary to keep.

**Review by breautek — APPROVED** [review]
lgtm

**Pull request merged**